seastar/src/submit_to.rs

```diff
@@ -34,12 +34,15 @@ mod ffi {
 ///     let ret = submit_to(0, || async { 42 }).await;
 ///     assert!(matches!(ret, 42));
 /// }
-pub async fn submit_to<Func, Fut, Ret>(shard_id: u32, func: Func) -> Ret
+/// ```
+pub fn submit_to<Func, Fut, Ret>(shard_id: u32, func: Func) -> impl Future<Output = Ret>
 where
     Func: FnOnce() -> Fut + Send + 'static,
     Fut: Future<Output = Ret> + 'static,
     Ret: Send + 'static,
 {
+    seastar::assert_runtime_is_running();
```

Comment:
I'm not sure why, but the build fails on this line. Perhaps `crate::assert_runtime_is_running()` would work instead?

Reply:
Indeed, it solves the issue.